src/decaylanguage/decay/viewer.py

```diff
@@ -186,25 +186,25 @@ def has_subdecay(ds: list[Any]) -> bool:
         iterate_chain(sc)
 
     @property
-    def graph(self) -> graphviz.dot.Digraph:
+    def graph(self) -> graphviz.Digraph:
```

Comment:
There is no Digraph in `graphviz.dot`. It's in `graphvis.graphs`, but it's also exported to the top level, so let's just use that one. This was also incorrect in the README.